File: URI/1040.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
	
		Scanner input = new Scanner(System.in);
		
		float a = input.nextFloat();
		float b = input.nextFloat();
		float c = input.nextFloat();
		float d = input.nextFloat();
		
		float media = (a * 2 + b * 3 + c * 4 + d * 1) / 10;
		System.out.printf("Media: %.1f%n", media);
		
		if(media >= 7.0) {
			System.out.println("Aluno aprovado.");
		}
		else if(media < 5.0) {
			System.out.println("Aluno reprovado.");
		}
		else {
			System.out.println("Aluno em exame.");
			
			float notaExame = input.nextFloat();
			System.out.printf("Nota do exame: %.1f%n", notaExame);
			
			media = (media + notaExame) / 2;
			if(media >= 5.0) {
				System.out.println("Aluno aprovado.");
			}
			else {
				System.out.println("Aluno reprovado.");
			}
			
			System.out.printf("Media final: %.1f%n", media);
		}
		
		input.close();

	}
	
}

File: URI/python/1040.py
# Questão 1040 em Python

grades = input().split()

n1, n2, n3, n4 = grades

average = (float(n1)*2 + float(n2)*3 + float(n3)*4 + float(n4))/10
print(f'Media: {average:.1f}')

if average >= 7.0:
    print('Aluno aprovado.')
elif average < 5.0:
    print('Aluno reprovado.')
elif 5.0 <= average <= 6.9:
    print('Aluno em exame.')
    exam = float(input())
    print(f'Nota do exame: {exam:.1f}')
    average = (average + exam)/2
    if average >= 5.0:
        print('Aluno aprovado.')
    elif average <= 4.9:
        print('Aluno reprovado.')
    print(f'Media final: {average:.1f}')